po/de.po

```diff
@@ -336,27 +336,27 @@ msgstr "keine Architektur"
 
 #: src/install.rs:138
 msgid "can't build package as root"
-msgstr "Pakete können nicht als root gebaut werden"
+msgstr "kann Pakete nicht als root bauen"
 
 #: src/install.rs:142
 msgid "failed to parse srcinfo generated by makepkg"
-msgstr "von Makepkg generierte srcinfo konnte nicht geparsed werden"
+msgstr "kann von Makepkg generiertes srcinfo nicht parsen"
```

Comment:
maybe `generierte` instead of `generiertes`  since `info` takes female gender in German

Reply:
oh, i see, this is a little unfortunate. das file, die datei, die info. in doubt i tend to use neutral, instead of female or male ...

Reply:
reverted all others you pointed out @mocdaniel - thanks!